File: src/main/java/architecture/ego_curios/events/ModEvents.java
package architecture.ego_curios.events;

import architecture.ego_curios.core.EGOCurios;
import architecture.resonator_combat_framework.api.event.AddGeckoLibCachePathEvent;
import net.neoforged.bus.api.SubscribeEvent;
import net.neoforged.fml.common.EventBusSubscriber;

@EventBusSubscriber(modid = EGOCurios.ID)
public final class ModEvents {
	@SubscribeEvent()
	public static void addGeckoLibCachePathEvent(AddGeckoLibCachePathEvent event) {
//		event.addAnimationPath(EGOCurios.modRlText("animation"));
//		event.addModelPath(EGOCurios.modRlText("geo_model"));
	}
}

File: src/main/kotlin/architecture/ego_curios/events/ModEvents.kt
package architecture.ego_curios.events

import architecture.ego_curios.core.EGOCuriosConstants
import architecture.resonator_combat_framework.event.AddGeckoLibCachePathEvent
import net.neoforged.bus.api.SubscribeEvent
import net.neoforged.fml.common.EventBusSubscriber

@EventBusSubscriber(modid = EGOCuriosConstants.ID)
object ModEvents {
	@SubscribeEvent
	fun addGeckoLibCachePathEvent(event: AddGeckoLibCachePathEvent) {
		// event.addAnimationPath(EGOCuriosConstants.modRlText("animation"))
		// event.addModelPath(EGOCuriosConstants.modRlText("geo_model"))
	}
}
